configs.py

```diff
@@ -37,6 +37,14 @@ def get_sys_exec_root_or_drive():
 if not all([algolia_app_id, algolia_api_key]):
     print("Application requires 'ALGOLIA_APP_ID' and 'ALGOLIA_API_KEY' for search")
 
+
+secret_key = os.environ.get('SECRET_KEY', 'change_secret_key')
```

Comment:
I'm not sure if my other comment is showing up weird for anyone else, so I'm going to comment again here. This `change_secret_key` and `'!@#$!!@$%@'` should not be in the `configs.py` file, it should be in the `.env` file. I would like these removed from the source code and put in that file, please. Thanks 😄